lib/elixir/lib/application.ex

```diff
@@ -523,10 +523,32 @@ defmodule Application do
     :application.set_env(app, key, value, opts)
   end
 
+  @doc """
+  Puts the environment for multiple apps at the same time.
+
+  It will fail if any of the applications appear more than once
+  or if any of the keys in the application are repeated.
```

Comment:
I think it would be clearer if it was written like

a) "or if repeated keys are found on an application config"
b) "or if any application contains repeated keys"
c) "or if any of the keys are repeated within an application"

On those suggestions I'm trying to make it clear that it's okay to have equal keys in different applications but not on the same applications :b